### PR TITLE
Juno development

### LINKED ISSUE #10: About Us Page - Juno
https://sites.google.com/codeustudents.com/summer-2019/week-1-about-us-page?authuser=0

### PR DESCRIPTION
**Pull request by Junochiu**
Close#10
First time making Pull Request,Let's see weather it works or not !

### CODE REVIEW

**Review by Sinyu104 — COMMENTED** [review]
Did you type `git pull origin master` into the terminal in order to get the most recent version of the master branch from the repo?
Just curious about the reason why the conflict happens. haha~

**Comment by Sinyu104** [comment]
> Did you type `git pull origin master` into the terminal in order to get the most recent version of the master branch from the repo?
> Just curious about the reason why the conflict happens. haha~

File: src/main/webapp/aboutus.html [hunk 1]
```diff
@@ -76,11 +76,14 @@ <h2>Teammate C Name</h2>
...
-<ul>
-    <li>Summer Feelz:</li>
-    <li>Aspirational Hobby:</li>
-    <li>Ask me About:</li>
-</ul>
+<h2>Juno Chiu</h2>
+    <ul>
+	  <li>Summer Feelz: Busy,hot,super hot!(I mean the weather)</li>
+      <li>Aspirational Hobby: Definitely, travel around the world! I'm always curious about different cultures and want to collect stories 
+	  through my practice. There's always something to be learnt in a travel! I want to go to every country, Arctic (and maybe bring some pengius back as pets XD like what Mr.Popper did)
```
Comment (ahuang13) [inline src/main/webapp/aboutus.html]:
```suggestion
	  through my practice. There's always something to be learnt in a travel! I want to go to every country, Arctic (and maybe bring some penguins back as pets XD like what Mr.Popper did)
```

**Review by ahuang13 — CHANGES_REQUESTED** [review]
It looks like some of the formatting is off. Can you try running "Reformat Code" in IntelliJ (assuming that's the IDE you're using).

**Review by 0xAlison — APPROVED** [review]

**Comment by Junochiu** [comment]
> reason why the conflict happened

yeah! I missed the step >< That's the reason why the conflict happened.

**Review by Sinyu104 — APPROVED** [review]

**Review by ahuang13 — APPROVED** [review]

**Pull request merged**